src/components/Tabs/TabsContext/TabsContext.scss

```diff
@@ -1,6 +0,0 @@
-@import "../../../styles/themes";
-@import "../../../styles/typography";
```

Comment:
These files have side-effects, are you absolutely sure it doesn't mess up anything?

Reply:
@sahariko it's imported in other places
![image](https://user-images.githubusercontent.com/2057934/141948558-14d1b9d6-6f62-431c-bd5b-b334d2ceddd4.png)
So I think it'd still have these effects. 

I can keep the import just in case.
the linter was annoyed about the empty block. it didn't care about the imports

Reply:
Let's keep it for now then.

Reply:
please keep the import as we support the option to only import a single file from the library - so the basic styles should be added

Reply:
@orrgottlieb I already put it the back, it's outdated